sandbox/modules/identity/src/main/java/org/opensearch/identity/support/ConfigHelper.java

```diff
@@ -162,4 +165,14 @@ public static <T> SecurityDynamicConfiguration<T> fromYamlString(
         return fromYamlReader(new StringReader(yamlString), ctype, version, seqNo, primaryTerm);
     }
 
+    public static String generateCommonLangPassword() {
+        String upperCaseLetters = RandomStringUtils.random(3, 65, 90, true, true);
+        String lowerCaseLetters = RandomStringUtils.random(3, 97, 122, true, true);
+        String numbers = RandomStringUtils.randomNumeric(4);
+        String totalChars = RandomStringUtils.randomAlphanumeric(3);
+        String combinedChars = upperCaseLetters.concat(lowerCaseLetters).concat(numbers).concat(totalChars);
+        List<Character> pwdChars = combinedChars.chars().mapToObj(c -> (char) c).collect(Collectors.toList());
+        String password = pwdChars.stream().collect(StringBuilder::new, StringBuilder::append, StringBuilder::append).toString();
```

Comment:
Can we shuffle these before returning the characters? Part of the security of having upper and lowercase is that increase the options for any given character. As is, this implementation will always have a set order so you don't end up with the increased order. Currently we have 26^3 + 26^3 + 10^4 + 62^3 but by shuffling we have 62^13.